index.js

```diff
@@ -498,37 +504,46 @@ MediaSession.prototype = extend(MediaSession.prototype, {
         this.removeStream(stream, true, cb);
     },
 
-    _removeRecvOnlySourceIfPresent: function(oldLocalDescription, newLocalDescription) {
-        var desc = oldLocalDescription;
 
-        // filter to only sources that changed from recvonly to sendrecv
-        desc.contents = desc.contents.filter(function(oldContent) {
-            if (oldContent.application.applicationType === 'rtp' && oldContent.application.sources && oldContent.application.sources.length) {
-                var newContent = findMatchingContentBlock(oldContent, newLocalDescription);
-                // the filter function handles the case where oldContent is null
-                return filterToMatchingRecvonly(oldContent, newContent);
-            }
-        });
-        delete desc.groups;
-        if (desc.contents.length > 0) {
-            this.send('source-remove', desc);
-        }
+
+    // Justin's new functions
```

Comment:
do we still need the comment? Or would a better description be more appropriate

Reply:
I like that comment!